### DISCUSSION TITLE
Using fauxmoESP with VS Code

### DISCUSSION THREAD

**Discussion by ray-leiter** [vintlabs/fauxmoESP#219] [General]
I used this excellent library to construct some code on an ESP32 that allows me to do on/off processing for some devices I have.
I did the development using Arduino IDE 1.8.19.
Does anyone know if this library can be used with VS Code (with the PlatformIO extension)?

**Comment by Haifeif** [comment]
I did it on Platformio, it worked until I kept adding devices. When I reached 14 devices, my system is nolonger workin
